code/game/objects/items/stacks/stack_recipes.dm

```diff
@@ -37,14 +37,16 @@
 /datum/stack_recipe/proc/finish_building(var/mob/usr, var/obj/item/stack/S, var/R) //This will be called after the recipe is done building, useful for doing something to the result if you want.
 	return R
 
-/datum/stack_recipe/proc/build(var/mob/usr, var/obj/item/stack/S, var/multiplier = 1)
+/datum/stack_recipe/proc/build(var/mob/usr, var/obj/item/stack/S, var/multiplier = 1, var/turf/construct_loc)
```

Comment:
>`usr` as a param
God what the fuck

Reply:
I noticed it too but it's used throughout. I just assumed bauds knew what he was doing 3 years ago and chose that for a very good reason

Reply:
@I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE

Reply:
I have no idea what this is. This is used by Topic() though so it always uses usr at the moment, if the parameter name causes problems just like rename it to user aka totally_not_usr like 99% of verbs and Topic() does

Reply:
`user` is the norm
And using this var name precludes the use of the actual `usr` var, and might potentially maybe even override it in procs it calls? Don't know about that one